### ISSUE TITLE
Build error: "error: ‘wl_display_connect’ was not declared in this scope"

### ISSUE THREAD

**Issue by Valmar33** [SaschaWillems/VulkanCapsViewer#170]
```
../VulkanCapsViewer-3.25_fixed/vulkancapsviewer.cpp: In member function ‘bool VulkanCapsViewer::initVulkan()’:
../VulkanCapsViewer-3.25_fixed/vulkancapsviewer.cpp:747:41: error: ‘wl_display_connect’ was not declared in this scope
  747 |             surfaceCreateInfo.display = wl_display_connect(NULL);
      |                                         ^~~~~~~~~~~~~~~~~~
```

**Comment by SaschaWillems** [comment]
Can you elaborate? What branch is "3.25_fixed"?

**Comment by firescry** [comment]
It seems that this problem was already addressed by this patch - 30be68d (error message provided in the commit message looks the same).

It was delivered as part of #155, which is already merged and available in version 3.28.

**Comment by Valmar33** [comment]
> Can you elaborate? What branch is "3.25_fixed"?

An Arch AUR package that apparently doesn't exist anymore... I guess it's woefully out-of-date, on that note.

Sorry for the issue. I'll close it now.
